docs/src/main/asciidoc/_client-guide/authentication-client.adoc

```diff
@@ -187,6 +192,8 @@ This element contains two attributes: -
 Java KeyStores also make use of a protection parameter when accessing a single entry in addition to the protection parameter to load the KeyStore, exactly one of the following elements must be present to specify the protection parameter of the entry being loaded.
 
  <key-store-clear-password password="..." />:: A password specified in the clear.
+ <key-store-masked-password algorithm="..." key-material="..." iteration-count="..." salt="..." masked-password="..." initialization-vector="..." />::
+  A password specified as encrypted.
  <credential-store-reference store="..." alias="..." clear-text="..." />:: Reference to a credential stored in a credential store.
```

Comment:
There are some XML snippets at the beginning of various sections in this doc that include a clear password element. It would be good to update these snippets to also include a masked password element.

Reply:
I think also double check to make sure that the masked password docs have been added in each section here where a clear password can currently be specified. Just noticed that it's missing from the ```<credentials>``` section.

Reply:
Fixed